packages/mobile/src/screens/edit-track-screen/fields/AccessAndSaleField/AccessAndSaleField.tsx

```diff
@@ -69,7 +73,7 @@ export const AccessAndSaleField = (props: AccessAndSaleFieldProps) => {
     if (isPremiumContentTipGated(premiumConditions)) {
       return [messages.specialAccess, messages.supportersOnly]
     }
-    if (isUnlisted) {
+    if (isUnlisted || isScheduledRelease) {
       return [messages.hidden, ...fieldVisibilityLabels]
     }
     return [messages.public]
```

Comment:
nit, isScheduledRelease needs to be in the hook deps or it breaks our lint :) (i fixed in my wip PR)

Reply:
sorry about that, why is CI green?

Reply:
great question.... our CI is horrible evidently. it used to block on this...